tools/lexicon.txt

```diff
@@ -24,6 +24,7 @@ aws
 backoff
 backoffdelay
 basedefs
+behaviour
```

Comment:
I think you can try behavior instead of behaviour and then you may not need to add it in lexicon as it is a valid dictionary word.

Reply:
I've updated it in 26b209a